FiveCalls/FiveCalls/IssuesViewController.swift

```diff
@@ -52,7 +49,7 @@ class IssuesViewController : UITableViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         // Construct viewModel from the issues already fetched by the caller.
-        viewModel = ViewModel(categories: self.issuesManager.categories)
+        viewModel = IssuesViewModel(categories: self.issuesManager.categories, showAll: shouldShowAllIssues)
```

Comment:
This is where we would build our view model instance.  Something like:

```swift
viewModel = shouldShowAllIssues ? CategorizedIssuesViewModel(issues: self.issuesManager.issues) : ActiveIssuesViewModel(issues: self.issuesManager.issues)
```

Since we're splitting this logic at the view model, it probably makes sense to push the categorization of issues back up into `IssuesManager` so we can hand the appropriate collection to the view model.